flixel/addons/editors/ogmo/FlxOgmo3Loader.hx

```diff
@@ -20,15 +20,10 @@ class FlxOgmo3Loader
 	var level:LevelData;
 
 	/**
-	 * Creates a new instance of `FlxOgmoLoader` and prepares the XML level data to be loaded.
```

Comment:
Oh, so the entire doc comment was inaccurate actually. :D

Reply:
hahaha yep 😅